### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by vicchig**

### CODE REVIEW

File: src/suggestions/suggestions-controller.ts [hunk 1]
```diff
@@ -456,8 +456,25 @@ export class MenuController {
...
+            //TODO: Very hacky way of fixing #569
+            //The issue is that the "no options" option is only added to the menu during the call to updateOptions
+            //if there is an attempt to create a menu that would have no options, the menu was simply not allowed to be created which is why the "no options" option was not visible any longer
+            if (suggestions.length === 0) {
+                suggestions.push(Actions.instance().actionsList[0]); //this does not have to be this specific aciton, just need one to create the option so that the menu is created and then we immediately delete the option
+            }
+
             const menu = this.module.menuController.buildMenu(suggestions, pos);
+
+            //TODO: Continuation of very hacky way of fixing #569
```
Comment (MajeedKazemi) [inline src/suggestions/suggestions-controller.ts:469]: is this going to be OK for the study?

Comment (MajeedKazemi) [reply src/suggestions/suggestions-controller.ts:469]: is it possible to cause any problems?

**Comment by MajeedKazemi** [comment]
looks good. will merge it now. Thanks @vicchig

File: src/suggestions/suggestions-controller.ts (hunk 1 above)
Comment (vicchig) [inline src/suggestions/suggestions-controller.ts:469]: This is actually fine, it is the code above that needs to be adjusted.

**Pull request merged**